### DISCUSSION TITLE
Can dynamoose be configured to not fail when it encounters NULL in string fields?

### DISCUSSION THREAD

**Discussion by smundro** [dynamoose/dynamoose#1591] [Q&A]
Getting a `Expected message to be of type string, instead found type null` message when retrieving data from dynamo, instead of doing something more friendly like returning `null` or `undefined` or `""`.  The `get()` method appears to run after the field validation, I'm not seeing any capacity for "or" types.  

 Is there any way to configure the dynamoose schema to handle null values?

**Comment by liam-ot** [comment]
I have only been able to implement this by setting the type of an attribute to `dynamoose.types.ANY`

This should be as simple as setting required to false, but for now this is a work around

Also put a `default` attribute value, for me it was also a String so an empty string value sufficed, this makes it more clear what the "real" type of the attribute is

**Comment by kevinlbatchelor** [comment]
ug right

**Comment by derekzernach** [comment]
Does setting type as `type: [String, dynamoose.type.NULL]` work? Thats what I'm doing.
